src/main/java/net/openhft/chronicle/core/util/ByteBuffers.java

```diff
@@ -27,10 +27,7 @@
  * Utility class for working with {@link ByteBuffer} instances.
  * <p>
  * Provides functionality to directly set the address and capacity of a ByteBuffer.
- * <p>
- * Note: This class is deprecated and will be removed in version x.26.
  */
-@Deprecated(/* to be removed x.26 */)
```

Comment:
why has this been undeprecated?

Reply:
The classes is needed in one place. So I have moved to that place on review.